### PR TITLE
Added pkgdown docs

### PR DESCRIPTION
**Pull request by austin3dickey**
Added a site we can put somewhere. Look at it at `r-pkg/uptasticsearch/docs/index.html`.

I also added a NEWS.md and README.md in the package's directory because they make rendering the site a lot easier. The new README is a copy of the top-level README with references to Python removed.

### CODE REVIEW

**Comment by terrytangyuan** [comment]
Looks good to me! BTW, I think we should probably remove the redundant `uptasticsearch` folder so we will only need uptasticsearch/r-pkg/ that contains the actual R package.

File: r-pkg/uptasticsearch/NEWS.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# uptasticsearch 0.1.0
+
+## First release!
```
Comment (terrytangyuan) [inline r-pkg/uptasticsearch/NEWS.md]: Maybe remove this line and the description for the package below. Put the other sub sections inside a section named like `Features`.

Comment (austin3dickey) [inline r-pkg/uptasticsearch/NEWS.md]: Did I make this change correctly?

Comment (terrytangyuan) [inline r-pkg/uptasticsearch/NEWS.md]: Yeah thanks!

**Pull request merged**